apps/frontend/src/app/5_pages/BitocracyPage/hooks/useProposalStatus.ts

```diff
@@ -7,23 +7,25 @@ import { decimalic } from '../../../../utils/math';
 import { ProposalState } from '../BitocracyPage.types';
 import { GRACE_PERIOD_IN_SECONDS } from '../components/Proposals/Proposals.constants';
 
-export const useProposalStatus = (proposal: Proposal) => {
+export const useProposalStatus = (proposal?: Proposal) => {
   const { value: blockNumber } = useBlockNumber();
   const currentTimeStamp = useMemo(() => Math.ceil(Date.now() / MS), []);
 
   const totalVotes = useMemo(
-    () => decimalic(proposal.votesFor).add(proposal.votesAgainst),
-    [proposal.votesFor, proposal.votesAgainst],
+    () => decimalic(proposal?.votesFor).add(proposal?.votesAgainst || 0),
+    [proposal],
   );
   const totalVotesMajorityPercentage = useMemo(
-    () => totalVotes.div(100).mul(proposal.majorityPercentage),
-    [totalVotes, proposal.majorityPercentage],
+    () => totalVotes.div(100).mul(proposal?.majorityPercentage || 0),
+    [totalVotes, proposal],
   );
```

Comment:
You don't need this as percentage, use `proposal.majorityPercentage` in line 35 direcly:
```
else if (
      decimalic(proposal.votesFor).lte(proposal.majorityPercentage) ||
      decimalic(totalVotes).lt(proposal.quorum)
    )
```